bokeh/charts/_chartobject.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, title, xlabel, ylabel, legend,
         self.__palette = palette
         self.__xgrid = xgrid
         self.__ygrid = ygrid
+        self.doc = doc
```

Comment:
I guess you don't want/need to provide chained method support for these attribute...